reporter/lib/events.js

```diff
@@ -0,0 +1,50 @@
+const io = require('socket.io-client');
+const { nanoid } = require('nanoid');
+
+const socket = io('http://localhost:5001/');
```

Comment:
Why are we connecting socket here? This will trigger a connect when you `require` this file.

Reply:
Yes, this is something that I want to discuss, would you suggest requiring it in  ```./reporter/index.js``` and then sending it to the event handlers and the event handlers would then return a function which would execute upon the triggering of the event

eg:
```js
const startHandler = (socket) => function(err, args) {
  // socket.emit('event-name');
}

newman.on('start', startHandler(socket));
```

Reply:
There's a better way, I think. This file can return a function. When you call that function, pass the socket and that function returns the events (by making a closure on the socket).